packages/wxt/package.json

```diff
@@ -183,6 +183,14 @@
       "types": "./dist/testing/index.d.ts",
       "default": "./dist/testing/index.mjs"
     },
+    "./testing/fake-browser": {
```

Comment:
I believe these should be placed above `./testing` so editors prefer them over the barrel file.

Reply:
🧠